standards_doc.md

```diff
@@ -0,0 +1,96 @@
+# STANDARDS DOC
+
+# DEPENDENCIES
+
+* Rails v5.2.2
+* Bundler
+* Ruby 2.6.1
+* [Homebrew](https://brew.sh/) 
+	* postgresql
+	* Suggested: rbenv or rvm
+* Bootstrap
+
+# HOW TO INSTALL DEPENDENCIES  
+
+## Install rails on your mac system
+Adding rails to your computer: [Mac Setup.pdf](https://www.dropbox.com/s/n9vxbzk47t8abq7/Mac%20Setup.pdf?dl=0)
+
+### Once Homebrew is installed
+
+~~~~
+brew tap homebrew/services
+~~~~
+
+You are going to need postgresql up and running
+~~~~
+brew services start postgresql
+~~~~
+~~~~
+brew install rbenv
+~~~~
+~~~~
+brew upgrade rbenv ruby-build
+~~~~
+~~~~
+rbenv init
+~~~~
+
+### Set Ruby local version for 'giving_profiles' project
+~~~~
+cd giving_profiles
+~~~~
+Check your ruby version
+~~~~
+ruby -v
+~~~~
+For this project you will need 2.6.1, if its not the listed version 
+~~~~
+export PATH="$HOME/.rbenv/shims:$PATH"
+~~~~
+~~~~
+rbenv install 2.6.1
+~~~~
+~~~~
+rbenv local 2.6.1
+~~~~
+
+# HOW TO SET UP THIS SITE FOR YOUR COMPUTER
+In the folder where you cloned the project:
```

Comment:
When you say `the folder` here, are you referring to the parent folder of giving profiles?

Reply:
I will reword this